corehq/apps/reports/standard/deployments.py

```diff
@@ -234,6 +248,44 @@ def process_rows(self, users, fmt_for_export=False):
             ])
         return rows
 
+    def process_users(self, users, fmt_for_export=False):
+        rows = []
+        for user in users[self.pagination.start:self.pagination.end]:
+            rows.append([
+                user_display_string(user.user_dim.username,
+                                    user.user_dim.first_name,
+                                    user.user_dim.last_name),
+                _fmt_date(user.last_form_submission_date, fmt_for_export),
+                _fmt_date(user.last_sync_log_date, fmt_for_export),
+                self.get_app_name(user.app_dim.app_id) or '---',
```

Comment:
we should add select_related to the queryset down below for the user_dim and app_dim records